Tribler/Test/Core/Video/test_vod.py

```diff
@@ -100,7 +99,7 @@ def state_callback(self, ds):
             lastsize = 1
             self.stream_read(stream, lastoff, lastsize, self.piecelen)
 
-            self.test_deferred.callback(None)
+            reactor.callFromThread(lambda: self.test_deferred.callback(None))
```

Comment:
This runs in a thread?

Reply:
At some point it was necessary to fix the test. I have just retested it now, it works without `callFromThread`. So, I'll remove it.

Reply:
Cool 👍